fjord/analytics/views.py

```diff
@@ -191,19 +193,33 @@ def dashboard(request):
 
     # Note: If we add additional querystring fields, we need to add
     # them to generate_dashboard_url.
-    search_happy = request.GET.get('happy', None)
-    search_platform = request.GET.get('platform', None)
-    search_locale = request.GET.get('locale', None)
-    search_product = request.GET.get('product', None)
-    search_version = request.GET.get('version', None)
+    if request.GET:
+        form = LeftDashboard(request.GET)
+        if form.is_valid():
+            search_happy = form.get('happy')
```

Comment:
You probably shouldn't be using these variables anymore, and instead access them all from form directly.